currency-conversion/src/main/java/lk/ac/ucsc/currencyconversion/CurrencyConversionBean.java

```diff
@@ -0,0 +1,83 @@
+package lk.ac.ucsc.currencyconversion;
+
+public class CurrencyConversionBean {
```

Comment:
@Binurim if possible do not use 'bean' in class names.

Reply:
Will look into this. Thank you!